decentraland/unity-renderer
# feat: friends ui polish

## What does this PR change?
This PR fix the next issues:

- [x] Update the friends row UI: quit the passport button. Move the jump in button, and quit the text, from the bottom of the player name and put it near the chat button instead for online players. 

Online friend
![Screenshot 2022-09-01 at 12.27.03.png](https://images.zenhubusercontent.com/5d91f7e9df37660001a11578/fcbc3d06-e55f-4bb6-9e32-9cdb8bf8cd14)

Offline friend
![Screenshot 2022-09-01 at 12.27.08.png](https://images.zenhubusercontent.com/5d91f7e9df37660001a11578/52518bf6-214d-4f19-ada8-e771f7c6be21)

- [x] The color of the icon must be #716B7C. Reduce the gap between it and the text.
![Screenshot 2022-09-01 at 18.06.01.png](https://images.zenhubusercontent.com/5d91f7e9df37660001a11578/d21927a7-72d7-4256-b24e-75053d86e73f):

- [x] If you open the [REQUESTS] tab, close the window, open it again and click on the [FRIENDS] tab, the list of friends is not being loaded correctly.
![image.png](https://images.zenhubusercontent.com/5eb2b8ca65c88f6e391ef16e/2b6f613f-6c32-41e4-8393-69e3ec065bf0)

## How to test the changes?
Go to https://play.decentraland.org/?renderer-branch=feat/friends-ui-polish and verify that all the issues above are fixed.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
